aero.py

```diff
@@ -638,7 +643,10 @@ def main():
 
     download_bundled()
 
-    if args.clean:
+    if args.only_run:
+        iso_path = os.path.join(BUILD_DIR, 'aero.iso')
```

Comment:
Here you should check if the iso file exists or not. If it doesn't, then it should build it first.